crates/rpc/rpc-eth-api/src/helpers/call.rs

```diff
@@ -807,6 +807,7 @@ pub trait Call: LoadState + SpawnBlocking {
             chain_id,
             blob_versioned_hashes,
             max_fee_per_blob_gas,
+            // authorization_list,
```

Comment:
is there an alloy issue for tracking 7702 rpc specs, that we can include alongside the `TransactionRequest` TODO?

Reply:
opened https://github.com/ethereum/execution-apis/issues/561
but we can go ahead and add rpc support on alloy, will track